### ISSUE TITLE
Pause video when searching for subtiltles.

### ISSUE THREAD

**Issue by mxswe** [boxeehacks/boxeehack#418]
It would be great if the movie youre watching would automaticly pause when searching for subtitles. XBMC have that feature and I really miss it in the boxeebox hack.

**Comment by Mike120** [comment]
Why not just pause it before you start searching for subtitles?

**Comment by mxswe** [comment]
I just think it's a nice feature to automaticly paus it when searching for subtitles. I always forget to paus it, and sometimes when I start rewind the movie the sub is out of sync because of that.

**Comment by boxeehacks** [comment]
It's a nice feature, sure... But it's also something which isn't really broken. So don't expect it to be too high on the priority list ;)

**Comment by mxswe** [comment]
Yeah that's true, but please add it to the priority list, doesn't need to be high prio but would be nice to see it in the future. Thank's for all the work you guys do for us!
